Neuron/Sections/Dapp/DappViewController.swift

```diff
@@ -29,8 +29,7 @@ class DappViewController: UIViewController, WKUIDelegate, WKNavigationDelegate,
 
     func didAddSubLayout() {
         webView = WKWebView.init(frame: CGRect(x: 0, y: 20, width: ScreenSize.width, height: ScreenSize.height - 49 - 20))
```

Comment:
* `init` is unnecessary.
* Hardcoding `20` (status bar height?) and `49` is bad smell and that won't adapt iPhone X screen. (But let's leave this as and discuss in next UI session)